core/src/autogluon/core/utils/utils.py

```diff
@@ -365,7 +365,7 @@ def generate_train_test_split(X: DataFrame,
                               problem_type: str,
                               test_size: float = 0.1,
                               random_state=0,
-                              min_cls_count_train=1) -> (DataFrame, DataFrame, Series, Series):
+                              min_cls_count_train=1) -> Tuple(DataFrame, DataFrame, Series, Series):
```

Comment:
I believe it should be `Tuple[]`
Ditto for the rest

Reply:
@yinweisu 

Thank you for pointing that out. I agree with your comment. I have fix it!